cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -62,7 +67,8 @@
     protected static final String VOLUME_MOUNT_HACK_NAME = "volume-mount-hack";
     private static final Long VOLUME_MOUNT_HACK_GROUPID = 1001L;
 
-    public static final String METRICS_CONFIG_FILE = "config.yml";
+    public static final String METRICS_CONFIG_FILE = "metrics-config.yml";
+    public static final String LOG_CONFIG_FILE = "log4j.properties";
```

Comment:
These aren't really files, they're keys in the ConfigMap, so I would name them `ANCILLARY_CM_KEY_METRICS` and `ANCILLARY_CM_KEY_LOG_CONFIG`, or something. (The common prefix helps to convey that they're grouped/related better than a common suffix would).